### PR TITLE
Add proper support for MultiPoint and fix broken MultiPolygon support

### PR DESCRIPTION
**Pull request by benhutchins**
Adding support for `MultiPoint` geometries.

Currently it is listed under `LIMITATIONS.md` as unsupported, but this didn't seem to make any sense since Google Maps supports [`google.maps.Data.MultiPoint`](https://developers.google.com/maps/documentation/javascript/reference/3.exp/data#Data.MultiPoint).

Added proper translation from OpenLayers to Google Maps `MultiPoint` and fixed the broken `MultiPolygon` support, which previously only converted a `MultiPolygon`'s first polygon into a single polygon. Using [`google.maps.Data.MultiPolygon`](https://developers.google.com/maps/documentation/javascript/reference/3.exp/data#Data.MultiPolygon) for that.

For testing, I created this simple GeoJSON example:

```json
{
"type": "FeatureCollection",
"features": [
{
   "type": "MultiPolygon",
   "coordinates": [
       [
           [ [102.0, 2.0], [103.0, 2.0], [103.0, 3.0], [102.0, 3.0], [102.0, 2.0] ]
       ],
       [
           [ [100.0, 0.0], [101.0, 0.0], [101.0, 1.0], [100.0, 1.0], [100.0, 0.0] ],
           [ [100.2, 0.2], [100.8, 0.2], [100.8, 0.8], [100.2, 0.8], [100.2, 0.2] ]
       ]
   ]
},
{
   "type": "MultiPoint",
   "coordinates": [
       [100.0, 0.0], [101.0, 1.0]
   ]
}
]
}
```

### CODE REVIEW

**Comment by adube** [comment]
Code is clean and build passes.  Excellent work!

Would you please add your example to the "concept" example, where the support of multi geometries are demonstrated?

I'd be able to quickly make a new release which would include all these.

Thank you very much.

**Comment by benhutchins** [comment]
@adube Thanks. I've added an example, a point for every island on Lake Superior.

They can also be dragged together, since it's one feature with many points.

**Comment by adube** [comment]
@benhutchins Excellent.  Thanks for the contribution.  Merging...

I'll create a release including these and report back here once it's done.

File: examples/concept.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <link rel="stylesheet" href="../css/ol3gm.css" type="text/css" />
     <link rel="stylesheet" href="./resources/layout.css" type="text/css" />
   </head>
-  <body>
+  <body>i
```
Comment (benhutchins) [inline examples/concept.html:13]: @adube This was a mistake, likely way jumping into iTerm, Cmd+Space…i… If you can remove this or I can do another PR.

Comment (adube) [reply examples/concept.html:13]: I'll remove it.

Comment (benhutchins) [reply examples/concept.html]: I'm also about to make a PR to fix an issue with `olgm.herald.Feature` not syncing styles, it honors styles if they're defined when a feature is added to a source, but it is not listening for the change to styles or to the `propertychange` event so it never updates the styles. This is breaking my use, where I only set styles to points after I have the image loaded, that was I can deal with the image failing to load and leave it as a default circle for the point.

Comment (adube) [reply examples/concept.html:13]: Indeed, the herald is not looking for these types of changes.  Okay, I'll wait.

Comment (benhutchins) [reply examples/concept.html:13]: Thanks @adube

**Pull request merged**